components/script/dom/some_func.rs

```diff
@@ -0,0 +1,78 @@
+// FIXME :: Rename this file...
```

Comment:
Since the only callers of these functions live in canvas_state.rs, we should move all of them into that file and remove this one.